### ISSUE TITLE
Examples are not working

### ISSUE THREAD

**Issue by rafaelramalho19** [vue-styleguidist/vue-styleguidist#41]
The /examples folder in this repository is not working out of the box.

I've run the instructions detailed in the Readme.md and there were errors in all components when serving vue-styleguidist:

```
JSDOC_ERROR: There are no input files to process.


Cannot parse src/components/RandomButton/RandomButton.vue: Error: JSDOC_ERROR: There are no input files to process.
```

**Comment by rafaesc** [comment]
Hi @rafaelramalho19,

I couldn't replicate the issue, do you follow the steps of https://github.com/vue-styleguidist/vue-styleguidist/tree/master/examples/basic? or do you run `npm run start`?

Also, you can build the example if you rename the file `.babelrc.lock` to `.babelrc`, and run `npm run start`

Please try it and let me know if it works.

**Comment by rafaelramalho19** [comment]
Hi @rafaesc, I've tried both your suggestions.

Both of them throw the same error.

If you open clone the project cleanly and run that script on your side, does it throw any error?

**Comment by rafaesc** [comment]
I tried that but even I couldn't replicate.

Here is the same error in "jsdoc-to-markdown":
https://github.com/jsdoc2md/jsdoc-to-markdown/iss

I think that vue-docgen-api need to [configure jsdoc](http://usejsdoc.org/about-configuring-jsdoc.html) then specify your config file path, that should solve the problem.

**Comment by rafaelramalho19** [comment]
The problem with that issue was that the folder had an underscore, which made jsdoc ignore the contents from the file. However it's not the case with me...

**Comment by rafaesc** [comment]
Please, can you add `console.log(sourceFiles);` in this path?

vue-styleguidist\node_modules\jsdoc-75lb\cli.js

on line 260. And run `npm run start`

In my case, print ` [ 'C:\\Users\\USER\\AppData\\Local\\Temp\\55io93ddkehk4pdcmztjk.js' ]`

**Comment by rafaelramalho19** [comment]
Hi @rafaesc sorry for the delayed response.


There's actually a path with an underscore in it, weird:

JSDOC_ERROR: [ '/var/folders/_4/f5fty0_10pjcvb6jyrcrkh600000gp/T/96xj6sca55kfm8mqvubhof.js' ]

**Comment by rafaesc** [comment]
Hi @rafaelramalho19,

I've published the fix under v1.3.2.

Regards,
